backend/inspirehep/search/facets.py

```diff
@@ -30,8 +30,24 @@ def inner(values):
     return inner
 
 
+def must_match_all_filter_nested(nested_path, match_field):
+    """Bool filter containing a list of must matches for nested queries."""
+
+    def inner(values):
+        if not values:
+            return
+        return Q(
+            "bool",
+            must=Q(
+                "nested", path=nested_path, query=Q("terms", **{match_field: values})
```

Comment:
is this going to work if there are several `values`, i.e. if multiple checkboxes are ticket in the facet? from the difference with `must_match_all_filter`, I strongly suspect it won't.